x-pack/plugins/observability/server/assets/constants.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const SLO_COMPONENT_TEMPLATE_MAPPINGS_NAME = '.slo-observability.slo-mappings';
+export const SLO_COMPONENT_TEMPLATE_SETTINGS_NAME = '.slo-observability.slo-settings';
+export const SLO_INDEX_TEMPLATE_NAME = 'observability-slo-data';
+export const SLO_INGEST_PIPELINE_NAME = 'observability-slo-monthly-index-default';
```

Comment:
We should set `default` to the same as the Kibana namespace when these are installed

Reply:
Does it mean not having it statically defined here?

Reply:
@maryam-saeidi yes that's correct. There is a `spaces` plugin that can return the current kibana namespace coming from a KibanaRequest. This will be wired in the route handler, so I'll just update the ResourceInstaller to take a spaceId as parameter.